modules/service/src/main/scala/lucuma/odb/service/ItcService.scala

```diff
@@ -188,8 +188,8 @@ object ItcService {
   ) extends TargetResult
 
   case class AsterismResult private (
-    acquisitionResult: Zipper[TargetResult],
-    scienceResult: Zipper[TargetResult]
+    acquisitionResult: Zipper[TargetImagingResult],
+    scienceResult: Zipper[TargetSpectroscopyResult]
```

Comment:
The more precise type here is required in order to pull out the "input" values below.